### ISSUE TITLE
The iteration for training network

### ISSUE THREAD

**Issue by PangzeCheung** [RenYurui/Global-Flow-Local-Attention#23]
The default niter is 5000000. I want to train the net on fashion dataset. How to set the parameters niter and niter_decay for training the PoseFlowNet and PoseModel? Thank you very much!

**Comment by RenYurui** [comment]
We use the default parameters. The learning rate did not change during training. 
We did not train the model for 5M iters. 
We just trained it until the loss converged. (both the flow fields estimator and the whole model)

**Comment by JialeTao** [comment]
> We use the default parameters. The learning rate did not change during training.
> We did not train the model for 5M iters.
> We just trained it until the loss converged. (both the flow fields estimator and the whole model)

Hi, I just met the same situation. Noticed that it will take very much time  to train the model for 5M iters. How many iters do you usually train the model in experience and how much time it takes?  In addition, how to judge the converged loss value of those terms?
